fenix/app/src/main/java/org/mozilla/fenix/ext/TopSite.kt

```diff
@@ -4,17 +4,30 @@
 
 package org.mozilla.fenix.ext
 
+import androidx.annotation.VisibleForTesting
 import mozilla.components.feature.top.sites.TopSite
 import org.mozilla.fenix.settings.SupportUtils
 
+@VisibleForTesting
+internal const val DEFAULT = "DEFAULT"
+
+@VisibleForTesting
+internal const val FRECENT = "FRECENT"
+
+@VisibleForTesting
+internal const val PINNED = "PINNED"
+
+@VisibleForTesting
+internal const val PROVIDED = "PROVIDED"
+
 /**
  * Returns the type name of the [TopSite].
  */
 fun TopSite.name(): String = when (this) {
-    is TopSite.Default -> "DEFAULT"
-    is TopSite.Frecent -> "FRECENT"
-    is TopSite.Pinned -> "PINNED"
-    is TopSite.Provided -> "PROVIDED"
+    is TopSite.Default -> DEFAULT
+    is TopSite.Frecent -> FRECENT
+    is TopSite.Pinned -> PINNED
+    is TopSite.Provided -> PROVIDED
```

Comment:
alternatively, you could just make these strings as a member of each enum, rather than keep the strings separate and then need `Topsite.name()` on top of it.

Reply:
Made the suggested changes